### DISCUSSION TITLE
LDAP Authentication

### DISCUSSION THREAD

**Discussion by sdeshpande** [fastapi/fastapi#2602] [Questions] [answered]
Hello Everyone,

Can anyone please tell me if it is possible to use "LDAP Authentication" with FastAPI?

Regards,
Saurabh

**Comment by Mause** [comment, marked as answer]
Yes, it is possible, but I don't know of any libraries that allow you to do it out of the box, you may have to implement it yourself

**Reply by sdeshpande** [reply]
Can you please point me in a direction ? I am pretty new to this but really interested in exploring. 
Thank you in advance.

**Reply by Mause** [reply]
https://github.com/noirello/bonsai

**Reply by sdeshpande** [reply]
Thank you. I will take a look.

**Reply by Turall** [reply]
@sdeshpande try my new module https://pypi.org/project/fastapi-ldap
